### PR TITLE
Add Zuul extra jobs

### PR DESCRIPTION
**Pull request by ssbarnea**
Migrate two other jobs from travis:
- tox-distros
- tox-devel

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/304

### CODE REVIEW

**Comment by ssbarnea** [comment]
recheck

File: zuul.d/layout.yaml [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+    vars:
+      tox_envlist: devel
+
+- job:
+    name: ansible-tox-distros
+    parent: ansible-tox-molecule
+    vars:
+      tox_envlist: distros
+    roles:
+      - zuul: install-docker
```
Comment (pabelanger) [inline zuul.d/layout.yaml]: This should be the project where install-docker is defined, so in this case opendev.org/zuul/zuul-jobs. However, we include this by default.

https://zuul-ci.org/docs/zuul/user/config.html#attr-job.roles.zuul

File: zuul.d/layout.yaml [hunk 2]
```diff
@@ -1,15 +1,23 @@
 ---
 # zuul.d/layout.yaml
+- job:
+    name: ansible-tox-devel
+    parent: ansible-tox-py36
+    vars:
+      tox_envlist: devel
+
+- job:
```
Comment (pabelanger) [inline zuul.d/layout.yaml:9]: what you need to do here, is add yourself a pre-run playbook, to ansible playbook that includes the install-docker role from zuul jobs.

File: bindep.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# This is a cross-platform list tracking distribution packages needed by tests;
```
Comment (pabelanger) [inline bindep.txt:1]: the other idea, is add docker here then us tools/test-setup.sh to ensure docker is started properly.

https://opendev.org/windmill/ansible-role-nodepool/src/commit/4666971ac5c2d2ac612aa085a002c1db60be09f6/tools/test-setup.sh

**Comment by ssbarnea** [comment]
recheck

**Pull request merged**